watchfaces/001-analog-70s-classic.qml

```diff
@@ -0,0 +1,349 @@
+            ctx.stroke()
+            ctx.closePath()
+        }
+    }
+
+    Canvas {
+        z: 2
+        id: secondHand
+        property var second: 0
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+            ctx.reset()
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.7)
+            ctx.shadowOffsetX = 1
+            ctx.shadowOffsetY = 1
+            ctx.shadowBlur = 2
+            ctx.strokeStyle = "red"
+            ctx.lineWidth = parent.height*0.008
+            ctx.beginPath()
+            ctx.moveTo(parent.width/2, parent.height/2)
+            ctx.lineTo(parent.width/2+Math.cos((second - 45)/60 * 2 * Math.PI)*width*0.1,
+                    parent.height/2+Math.sin((second - 45)/60 * 2 * Math.PI)*width*0.1)
+            ctx.stroke()
+            ctx.closePath()
+            ctx.beginPath()
+            ctx.fillStyle = "red"
+            ctx.arc(parent.width/2, parent.height/2, parent.height*0.012, 0, 2*Math.PI, false)
+            ctx.fill()
+            ctx.moveTo(parent.width/2, parent.height/2)
+            ctx.lineTo(parent.width/2+Math.cos((second - 15)/60 * 2 * Math.PI)*width*0.32,
+                    parent.height/2+Math.sin((second - 15)/60 * 2 * Math.PI)*width*0.32)
+            ctx.stroke()
+            ctx.closePath()
+
+        }
+    }
+
+    Canvas {
+        z: 11
+        id: nailDot
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+            ctx.reset()
+            ctx.beginPath()
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.8)
+            ctx.shadowOffsetX = 1
+            ctx.shadowOffsetY = 1
+            ctx.shadowBlur = 1
+            ctx.fillStyle = Qt.rgba(1, 1, 1, 1)
+            ctx.arc(parent.width/2, parent.height/2, parent.height*0.006, 0, 2*Math.PI, false)
+            ctx.fill()
+            ctx.closePath()
+        }
+    }
+
+    Canvas {
+        z: 3
+        id: hourStrokes
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+
+            ctx.lineWidth = parent.width*0.025
+            ctx.strokeStyle = Qt.rgba(1, 1, 1, 0.9)
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.8)
+            ctx.shadowOffsetX = 0
+            ctx.shadowOffsetY = 0
+            ctx.shadowBlur = 2
+            ctx.translate(parent.width/2, parent.height/2)
+            for (var i=0; i < 12; i++) {
+                if ((i%3) == 0) {
+
+                    ctx.beginPath()
+                    ctx.moveTo(0, height*0.36)
+                    ctx.lineTo(0, height*0.46)
+                    ctx.stroke()
+                }
+                ctx.rotate(Math.PI/6)
+            }
+        }
+    }
+
+    Canvas {
+        z: 3
+        id: min5Strokes
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
```

Comment:
I noticed you change this a little bit from what you had uploaded before https://github.com/AsteroidOS/unofficial-watchfaces/blob/master/.thumbnails/analog-70s-classic.jpg
I would like those shorter "min5Strokes" to be thinner as well. Having them the same size as the four hourStrokes gives a weird feeling

Reply:
agreed.

Reply:
It is better indeed!